jit/analyze.ts

```diff
@@ -90,7 +90,7 @@ module J2ME {
       return true;
     }
     // XXX Determine whether we can start using the code in this function.
-    return false;
+    return methodInfo.isFinal;
     var result = methodInfo.isFinal;
```

Comment:
Could we now use the code in this function? Maybe it wasn't working before a1fd332 and it's working now?